docs/wiki/Deploying-ALZ-Pre-requisites.md

```diff
@@ -54,3 +54,13 @@ New-AzRoleAssignment -Scope '/' -RoleDefinitionName 'Owner' -ObjectId $user.Id
 ```
 
 > Please note: sometimes it can take up to 15 minutes for permission to propagate at tenant root scope. It is highly recommended that you log out and log back in to refresh the token before you proceed with the deployment.*
+
+### Azure Monitor Baseline Alerts prerequisites
+
+The Azure Monitor Baseline Alerts are deployed as part of the Enterprise-Scale deployment, and they require the following:
+
+1. For the policies to work, the following Azure resource providers, normally registered by default, must be registered on all subscriptions in scope:
+  - Microsoft.AlertsManagement
+  - Microsoft.Insights
+Please see [here](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider) for details on how to register a resource provider should you need to do so.
+2. For leveraging the log alerts for Virtual Machines, ensure that VM Insights is enabled for the Virtual Machines to be monitored. For more details on VM Insights deployment see [here](https://learn.microsoft.com/en-us/azure/azure-monitor/vm/vminsights-enable-overview). Please note only the performance collection of the VM insights solution is required for the current alerts to deploy.
```

Comment:
Good point. The "Legacy - Enable Azure Monitor for VMs" actually does this, so if the portal accelerator is deployed with default settings it will be handled. Should I remove this bullet or make a reference to the policy?

Reply:
Removed bullet 2